pkg/recordset/stack_template.go

```diff
@@ -200,7 +201,21 @@ func (m *Manager) getEniList(clusterID string, baseDomain string) ([]EtcdEni, er
 		return nil, microerror.Mask(err)
 	}
 
-	for i, nic := range output.NetworkInterfaces {
+	nicList := output.NetworkInterfaces
+
+	fmt.Printf("unsorted nic list: ")
+	for _, nic := range nicList {
+		fmt.Printf("%s ", getNICNameFromTag(nic.TagSet))
+	}
```

Comment:
Do we want to keep the printing, I'd assume if ordering is working there's no need to print before and after, just to avoid the loops.

If it's needed why not using logging? Isn't it implemented?

Reply:
this is just for debug to see if this work as expected, it will be removed

I hate that GitHub request reviews right away without me being ready :(

Reply:
Oh sorry 🤦🏻 my bad, maybe next time you can create it as a draft so it won't ping. That's at least what I'm doing most of the time 🙂